pkgs/by-name/fa/falkor/package.nix

```
@@ -0,0 +1,56 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchurl,
+  electron,
+  dpkg,
+  makeWrapper,
+  commandLineArgs ? "",
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "falkor";
+  version = "0.0.92";
+
+  src = fetchurl {
+    url = "https://github.com/Team-Falkor/app/releases/download/v${finalAttrs.version}/falkor.deb";
+    hash = "sha256-yDpYu2ehrRQuD29jcyTQla2R2IT1zfBDeWDDRnmqc8Y=";
+  };
+
+  unpackPhase = ''
+    runHook preUnpack
+    dpkg -x $src ./
+    runHook postUnpack
+  '';
+
+  nativeBuildInputs = [
+    makeWrapper
+    dpkg
+  ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin
+    mv usr/share $out/share
+    sed -i "s|Exec=.*|Exec=$out/bin/falkor|" $out/share/applications/*.desktop
```

```suggestion
    sed -i "s|Exec=.*|Exec=falkor|" $out/share/applications/*.desktop
```
Absolute paths can become outdated if your desktop environment copies the .desktop file out of the store.
Using just the executable name should be better, since that's always on the path anyways.